repository/RediStick-Core.package/RsRediStick.class/instance/connect.st

```diff
@@ -0,0 +1,4 @@
+actions
+connect
```

Comment:
in my opinion, I think when we connect a stick it authenticate himself

Reply:
The API would be simpler. However, there is a case that you do not need to authenticate.
So I think it would be better to call #shoudAuthenticate testing method explicitly.
```
connect
  super connect.
  self shoudAuthenticate ifTrue: [self authenticate].

shoudAuthenticate
  ^ self password notNil

authenticate
  self userName ifNil: [ ^ self endpoint auth: self password ]
  self endpoint auth: self password user: self userName ]
```

And 'authentificate' is a typo. 'Authentification' is valid as a noun, but as a verb, 'authentificate' is not common.

Reply:
thank you for this great reviewing, I learn a lot! 
you're right it's looking more sophisticated than my version. 
I will fix this in a next commit

Reply:
fix with commit 
27c5443